packages/dart/npt_flutter/linux/CMakeLists.txt

```diff
@@ -43,7 +44,11 @@ function(APPLY_STANDARD_SETTINGS TARGET)
   target_compile_features(${TARGET} PUBLIC cxx_std_14)
   target_compile_options(${TARGET} PRIVATE -Wall -Werror)
   target_compile_options(${TARGET} PRIVATE "$<$<NOT:$<CONFIG:Debug>>:-O3>")
-  target_compile_definitions(${TARGET} PRIVATE "$<$<NOT:$<CONFIG:Debug>>:NDEBUG>")
+  target_compile_definitions(
+    ${TARGET}
+    PRIVATE "$<$<NOT:$<CONFIG:Debug>>:NDEBUG>"
+  )
+  target_compile_options(${TARGET} PRIVATE -Wno-error=deprecated-declarations) # Allow deprecated warnings for tray_manager package
```

Comment:
This line is added to allow builds despite usage of a deprecated libadwaita api in the tray_manager plugin.